src/main/java/me/tomsdevsn/hetznercloud/exception/APIRequestException.java

```diff
@@ -0,0 +1,15 @@
+package me.tomsdevsn.hetznercloud.exception;
+
+import lombok.Getter;
+import me.tomsdevsn.hetznercloud.objects.response.APIErrorResponse;
+
+public class APIRequestException extends RuntimeException {
```

Comment:
You can use `@StandardException` as annotation to create an exception pretty easy (I'll change that for the other exceptions in my current draft mr).

Reply:
This feature is experimental, I wouldn't recommend using it at this stage.
I need to write the standard constructors anyways because I want to wrap the APIError(Response), so this doesn't seem to benefit me.